### PR TITLE
React-Week1 hiren

### PR DESCRIPTION
**Pull request by hirenkumar91**
Dear Mentor Here is resubmission of my home work

### CODE REVIEW

File: app/about_us/aboutCSS/crew.module.css
Comment (vdaguenet) [inline app/about_us/aboutCSS/crew.module.css:1]: Be careful with the indentation in this file

File: app/about_us/aboutCSS/partners.module.css
Comment (vdaguenet) [inline app/about_us/aboutCSS/partners.module.css:1]: Mind the indentation

File: app/about_us/ourCrew.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        imageUrl: "./crew/image-anousheh-ansari.png",
+        description:
+          "Maya brings a unique blend of technical skills and customer service experience to the team. She’s always ready to assist with any needs and to make sure every traveler has an unforgettable experience.",
+      },
+];
+
+const CrewDetails =({ name, title,imageUrl,description}) =>{
+    return(
+    <div className={styles.crewCard}>
+    <div class={styles.cardImg}>
```
Comment (vdaguenet) [inline app/about_us/ourCrew.js]: Remember to index your JSX code. The rules we use for indenting HTML also apply to JSX.

File: app/about_us/ourCrew.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        </p>
+    </div>
+    </div>
+    );
+};
+
+
+export default function OurCrew() {
+    return (
+      < div className={styles.crewContainer}>
```
Comment (vdaguenet) [inline app/about_us/ourCrew.js]:
```suggestion
      <div className={styles.crewContainer}>
```

File: components/ui/Footer.js [hunk 3]
```diff
@@ -15,20 +15,21 @@ export const Footer = () => {
...
       {/* Docs for the Link: https://nextjs.org/docs/pages/api-reference/components/link */}
-
-      {/* TASK - React 1 week 1 */}
-      {/* Add a new list item for LINKEDIN */}
+      
       <div className={styles.footerLinks}>
         <h3>Follow us</h3>
         <ul className={styles.footerList}>
+        <li>
+            <a href='https://linkedin.com'>linkedin</a>
```
Comment (vdaguenet) [inline components/ui/Footer.js]: Mind the indentation here too

**Review by vdaguenet — COMMENTED** [review]
Remember the indentation rules you have learned so far. They are also valid for React!

File: app/about_us/aboutCSS/crew.module.css
Comment (hirenkumar91) [reply app/about_us/aboutCSS/crew.module.css:1]: code  indentation done

File: app/about_us/aboutCSS/partners.module.css
Comment (hirenkumar91) [reply app/about_us/aboutCSS/partners.module.css:1]: code indentation done

File: app/about_us/ourCrew.js (hunk 1 above)
Comment (hirenkumar91) [reply app/about_us/ourCrew.js]: code indentation done

File: app/about_us/ourCrew.js (hunk 2 above)
Comment (hirenkumar91) [reply app/about_us/ourCrew.js]: code indentation done

File: components/ui/Footer.js (hunk 3 above)
Comment (hirenkumar91) [reply components/ui/Footer.js]: code indentation done

**Comment by hirenkumar91** [comment]
Dear Mentor all suggested updates done should i merge branch

**Review by vdaguenet — APPROVED** [review]

**Pull request merged**